src/assets/icons/icon-trash-dark.svg

```diff
@@ -0,0 +1,6 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" viewBox="0 0 16 16">
+  <g fill="#32363f" fill-rule="evenodd" transform="translate(-96 -32)">
```

Comment:
https://medium.com/@david.gilbertson/icons-as-react-components-de3e33cb8792#.xzmogaxo8
What do you think of this solution?